cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -43,6 +43,9 @@ This section explains how to use Cypher to manage Neo4j role-based access contro
 *** <<administration-security-administration-dbms-custom, Creating custom roles with DBMS privileges>>
```

Comment:
The different toc seems to not be completely synced for the DBMS privileges, please sync them all with how the section is structured